web/portal/templates/doc/cosmodc2.md

```diff
@@ -1,21 +1,36 @@
 <!--- Do not delete this line, it is needed for jinja_markdown to render this page correctly -->
```

Comment:
The comments have started to appear on the rendered page.  Either we need to find a way to prevent that or remove the line from all the .md files.  See: http://lb.desc-web-dev.production.svc.spin.nersc.org/doc/dc2_sim_sky_survey